conda/gateways/repodata/jlap/fetch.py

```diff
@@ -374,7 +374,10 @@ def request_url_jlap_state(
                     cache=cache,
                     temp_path=temp_path,
                 )
-            log.exception("Requests error")
```

Comment:
This error will happen when the remote file rolls over and our local file position is outside the range of the remote file. Or if it happens frequently, you might want to know that the remote doesn't support range requests.